### PR TITLE
Readme improvement for v2023.10 release

### PR DESCRIPTION
**Pull request by arnopo**
Update documentation reference and wrap long lines.

### CODE REVIEW

**Review by wmamills — COMMENTED** [review]
Hopefully you mean "more than 100 characters"

I hope we did not have lines with more than 1000 :)

File: README.md [hunk 1]
```diff
@@ -312,4 +312,5 @@ We use standard github mechanism for pull request. Please refer to github docume
 ## Communication and Collaboration
 [Subscribe](https://lists.openampproject.org/mailman3/lists/openamp-rp.lists.openampproject.org/) to the OpenAMP mailing list(openamp-rp@lists.openampproject.org).
 
-For more details on the framework please refer to the the [OpenAMP wiki](https://github.com/OpenAMP/open-amp/wiki).
+For more details on the framework please refer to the the
```
Comment (edmooring) [inline README.md:327]: "the the" should be just "the" at the end of the line.

**Review by edmooring — CHANGES_REQUESTED** [review]
Otherwise looks good.

**Review by tnmysh — APPROVED** [review]
Approved assuming Ed's comment will be fixed.

File: README.md (hunk 1 above)
Comment (edmooring) [reply README.md:327]: If it holds up the release, we can fix it later.

Comment (arnopo) [reply README.md]: Good catch! I fixed it

**Pull request merged**